javascript/javascript1/week1/ez-namey.js

```diff
@@ -1,5 +1,4 @@
-let firstWords = ["Easy", "Nice", "Amazing", "Cool", "Fantastic", "Innovative", "Revolutionary", "Futuristic", "AI-driven", "Successful"];
-let secondWords = ["Company", "Firm", "Corporation", "Startup", "Team", "Idea", "Aim", "Job", "Project", "Innovation"];
-const randomNumber = Math.floor(Math.random() * 10);
-let startupName = firstWords[randomNumber] + " " + secondWords[randomNumber];
-console.log ("The startup: " + JSON.stringify(startupName) + " contains " + startupName.length + " characters")
+const firstWords = ["Easy", "Nice", "Amazing", "Cool", "Fantastic", "Innovative", "Revolutionary", "Futuristic", "AI-driven", "Successful"];
+const secondWords = ["Company", "Firm", "Corporation", "Startup", "Team", "Idea", "Aim", "Job", "Project", "Innovation"];
+const startupName = `${firstWords[Math.floor(Math.random() * 10)]} ${secondWords[Math.floor(Math.random() * 10)]}`;
+console.log (`The startup: ${JSON.stringify(startupName)} contains ${startupName.length} characters`)
```

Comment:
why do you need JSON.stringify?

Reply:
to add quotation marks around startup name, e.g. "Awesome Company"